src/course-outline/card-header/CardHeader.jsx

```diff
@@ -61,6 +61,18 @@ const CardHeader = ({
   const [titleValue, setTitleValue] = useState(title);
   const cardHeaderRef = useRef(null);
   const [isManageTagsDrawerOpen, openManageTagsDrawer, closeManageTagsDrawer] = useToggle(false);
+  let fullProctoringExamConfigurationLink = () => proctoringExamConfigurationLink;
+
+  // check if proctoringExamConfigurationLink is a relative link
+  if (proctoringExamConfigurationLink && !(
+    proctoringExamConfigurationLink.indexOf('http://') === 0 || proctoringExamConfigurationLink.indexOf('https://') === 0
+  )) {
+    // new URL needs to called inside a function for tutor
+    fullProctoringExamConfigurationLink = () => new URL(
+      proctoringExamConfigurationLink,
+      getConfig().STUDIO_BASE_URL,
+    ).href;
+  }
```

Comment:
I this this code is more complex than it needs to be. 

```suggestion
  const fullProctoringExamConfigurationLink = () => (
      proctoringExamConfigurationLink && new URL(proctoringExamConfigurationLink, getConfig().STUDIO_BASE_URL.href;
    );
```

If the link is defined, and is relative then this will make the url relative to the studio base. If it's not a relative url it will override the base url.

Reply:
I'm also not sure why this needs to be a function, nothing that runs in it should change between the component being mounted and the function being called.

Reply:
@xitij2000 Thanks! updated.

> I'm also not sure why this needs to be a function, nothing that runs in it should change between the component being mounted and the function being called.

Same here. @KristinAoki Do we have a link or a PR which explains this tutor related bug?

Reply:
https://github.com/openedx/frontend-app-course-authoring/pull/706